src/app/query/stacks/nonce/account-nonces.hooks.ts

```diff
@@ -16,6 +16,7 @@ export function useNextNonce(address: string) {
 
   return useQuery({
     ...createGetAccountNoncesQueryOptions({ address, client, network: network.chain.stacks.url }),
+    staleTime: 0,
```

Comment:
I seem to recall if you set this to `Infinity` it will never fetch from the cache - which is perhaps what you're aiming to do with the `useEffect` essentially?

Reply:
Sorry, that does the inverse of what I was thinking - my bad.

Reply:
We have an almost identical version of this hook in mobile. We should consolidate them if we can